format/spec.md

```diff
@@ -702,6 +703,21 @@ Blob metadata is a struct with the following fields:
 | _optional_ | _optional_ | **`properties`** | `map<string, string>` | Additional properties associated with the statistic. Subset of Blob properties in the Puffin file. |
 
 
+#### Partition statistics
+
+Partition statistics files are the valid files based on [Partition statistics spec](../partition-statistics-spec). Partition statistics are informational. A reader can choose to
+ignore partition statistics information. Partition statistics support is not required to read the table correctly. A table can contain
+many partition statistics files associated with different table snapshots.
```

Comment:
It's fine to say that partition stats files are informational and it's okay not to read them. But what we really need is a statement of the specific _requirements_ for partition stats files. When producing them, what must be true for them to be reliable information for engines?

Reply:
Added some points. Please suggest if anything else needs to be added.

Reply:
I think what we need to add is that the stats must be accurate if provided, otherwise it will be hard to depend on this information.

Reply:
I have added now. 

Initially didn't add thinking it is implicit.